include/UIKit/UIKitTypes.h

```diff
@@ -48,7 +48,7 @@ typedef NS_ENUM(NSUInteger, NSWritingDirection) {
 };
 
 enum {
-    UITextAlignmentLeft,
+    UITextAlignmentLeft = 0,
     UITextAlignmentCenter,
```

Comment:
We had this as `NS_ENUM` before; it's somewhat silly but it makes it unable to be used itnerchangeably with NSTextAlignment.

---
In reply to: [85448628](https://github.com/Microsoft/WinObjC/pull/1254#discussion_r85448628) [](ancestors = 85448628)